### PR TITLE
[FRONT-117] Implement Docs article "Sharing resources in Core"

### PR DESCRIPTION
**Pull request by juhah**
New docs article for sharing resources in Core, about sharing resources (in Core).

### CODE REVIEW

File: app/src/app/Docs.jsx [hunk 1]
```diff
@@ -44,7 +45,7 @@ import BuildingCustomModule from '$docs/components/Pages/Tutorials/BuildingCusto
 import DataToken from '$docs/components/Pages/DataToken'
 // Core Docs
 import IntroToCore from '$docs/components/Pages/Core/IntroToCore'
-import UsingCanvasesInCore from '$docs/components/Pages/Core/UsingCanvasesInCore'
+import SharingResourcesInCore from '$docs/components/Pages/Core/SharingResourcesInCore'
```
Comment (juhah) [inline app/src/app/Docs.jsx:48]: Moved this under Canvases sections since it's there in the nav.

**Review by tumppi — APPROVED** [review]
Looks good to me 👍

File: app/src/docs/content/core/sharingResourcesInCore.mdx [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        <video muted loop autoPlay>
+            <source src={Sharing} type="video/mp4" />
+        </video>
+    </div>
+</figure>
+
+## The share sidebar
+
+Wherever you access it from, all resources are shared from the sidebar. The general flow 
+of sharing with the sidebar works like this.
```
Comment (fonty1) [inline app/src/docs/content/core/sharingResourcesInCore.mdx:29]: of sharing with the sidebar works like this.
```suggestion
of sharing with the sidebar works like this:
```

File: app/src/docs/content/core/sharingResourcesInCore.mdx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+1. Choose private or public resource
+2. Type the address of the person you want invite to share (either email or Ethereum)
```
Comment (fonty1) [inline app/src/docs/content/core/sharingResourcesInCore.mdx:32]: Are we still including emails?

File: app/src/docs/content/core/sharingResourcesInCore.mdx [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+3. Choose the permissions for that user, either via the role chooser or build a custom role by clicking the checkboxes
+4. Add, remove or adjust permissions for as many people as you want
+5. Click Save to finalise any changes button before you close the side bar
```
Comment (fonty1) [inline app/src/docs/content/core/sharingResourcesInCore.mdx:35]: 5. Click Save to finalise any changes button before you close the side bar
```suggestion
5. Click Save to finalise any changes before you close the side bar
```

**Comment by fonty1** [comment]
@juhah could you add a reference section to this in `/docs/streams/using-streams-in-core` about the sharing? That's the main use case and they may only read that page of the docs.

Since Dashboards and Canvases will go in the next few months, this will need to be refactored to be 'stream sharing', etc but that's a battle for another day.

**Review by fonty1 — APPROVED** [review]

File: app/src/docs/content/core/sharingResourcesInCore.mdx (hunk 3 above)
Comment (juhah) [reply app/src/docs/content/core/sharingResourcesInCore.mdx]: No, I guess that was just old copy... changed it to say `Type the Ethereum address of the person you want invite to share`

**Comment by juhah** [comment]
>  could you add a reference section to this in /docs/streams/using-streams-in-core about the sharing? That's the main use case and they may only read that page of the docs.

Ok, added a mention to Streams, Canvases & Dashboard pages about sharing:
Screenshot 2021-02-22 at 14 49 43

**Pull request merged**